### PR TITLE
typo fix

### PR DESCRIPTION
**Pull request by Glusk2**

### CODE REVIEW

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this pull request

**Comment by 0crat** [comment]
@Glusk2 this pull request is too small, just 2 lines changed (less than 10), there will be no formal code review, see [§53](http://www.zerocracy.com/policy.html#53) and [§28](http://www.zerocracy.com/policy.html#28); in the future, try to make sure your pull requests are not too small; @yegor256[/z](https://www.0crat.com/u/yegor256) please review this and merge or reject

File: _posts/2015/jun/2015-06-29-simple-diagrams.md [hunk 1]
```diff
@@ -104,7 +104,7 @@ understand you much faster.
 
 **Don't Use Colors**.
 Or let me put it this way: Don't abuse colors. And in order to avoid abusing
-them, you be better off staying away from colors in the first place. If you need
```
Comment (degree) [inline _posts/2015/jun/2015-06-29-simple-diagrams.md]: "you'd be" seem also an option.

**Comment by yegor256** [comment]
@johndotpage what do you think?

**Comment by johndotpage** [comment]
@yegor256 Both suggestion are fine, I perhaps prefer "you'd be" though.

**Comment by 0crat** [comment]
Job `gh:yegor256/blog#412` is not assigned, can't get performer

**Comment by 0crat** [comment]
This job is not in scope

**Pull request merged**